python/llm/src/bigdl/llm/ggml/quantize.py

```diff
@@ -38,7 +39,7 @@
                           "q8_0": 7}
 
 _quantize_type = {"llama": _llama_quantize_type,
-                  "bloomz": _bloomz_quantize_type,
+                  "bloom": _bloom_quantize_type,
                   "gptneox": _gptneox_quantize_type}
 
 _valid_types = set(list(_llama_quantize_type.keys()) + list(_bloomz_quantize_type.keys()) +
```

Comment:
`_bloomz_quantize_type.keys()` => `_bloom_quantize_type.keys()`
Seems that you forgot to change here